app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -314,8 +314,10 @@ def parse_endpoint(entity)
       new_result
     end
 
-    def parse_namespaces(container_projects)
-      parse_base_item(container_projects).except(:namespace)
+    def parse_namespaces(namespace)
+      x = parse_base_item(namespace).except(:namespace).merge(
+        :labels => parse_labels(namespace)
+      )
```

Comment:
Oops, `x =` is a silly leftover from debugging.
Why does it even work?  Ah, yes, ruby assignments also return the value.
Will rewrite as Ari suggests and also try to add a test.